src/main/java/com/ebay/myriad/configuration/MyriadConfiguration.java

```diff
@@ -179,6 +202,14 @@ public Integer getZkTimeout() {
 
     public Integer getRestApiPort() { return this.restApiPort != null ? this.restApiPort : DEFAULT_REST_API_PORT; }
 
+    public String getWebuiURL() {
+        try {
+            return new URL( "http", getResourceManagerHost(), getRestApiPort(), "").toString();
```

Comment:
Why not just link to the YARN RM Web UI on port 8088 (default for yarn.resourcemanager.webapp.address)?